File: beginner/pagina01/1035/1035.lua
local a, b, c, d = io.read():match("([%-]?%d+) ([%-]?%d+) ([%-]?%d+) ([%-]?%d+)")
a, b, c, d = tonumber(a), tonumber(b), tonumber(c), tonumber(d)

if b > c
    and d > a
    and (c + d) > (a + b)
    and c >= 0
    and d >= 0
    and (a % 2 == 0)
then
    print("Valores aceitos")
else
    print("Valores nao aceitos")
end

File: beginner/__1000__/1035/1035.rs
use std::io;

fn main() {

    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    let data: Vec<i32> = input.trim().split_whitespace().map(|x| x.parse().unwrap()).collect();
    let (a, b, c, d) = (data[0], data[1], data[2], data[3]);

    if b > c && d > a && (c + d) > (a + b) && c >= 0 && d >= 0 && a % 2 == 0 {
		println!("Valores aceitos");
	} else {
		println!("Valores nao aceitos");
	}
}